libs/iota-browser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi-tdk/iota-browser",
-  "version": "0.2.4",
+  "version": "0.2.5",
```

Comment:
**note:** 
versioning has to be done by semantic-release

Reply:
yeah, we will reset before merging